bootloader/lkenv/lkenv_v2.go

```diff
@@ -114,6 +114,19 @@ type SnapBootSelect_v2_recovery struct {
 	 */
 	Recovery_system_status [SNAP_FILE_NAME_MAX_LEN]byte
 
+	/** device_lock_state contains the lock state of the device. It is used by the
+	 * bootloader to track device lock changes. When lock state changes, device goes
+	 * automatically to install mode. This entry is completely transparent
+	 * to the snapd and is only modified by bootloader.
+	 * Only first char in the aray is used (device_lock_state[0])
+	 * Permitted values:
+	 *  0: DEVICE_STATE_UNKNOWN:   initial value at first boot.
```

Comment:
super nitpick (look at the indentation of the following values):
```suggestion
	 *  0: DEVICE_STATE_UNKNOWN:  initial value at first boot.
```